tests/unit/test_activation_response.py

```diff
@@ -0,0 +1,98 @@
+import requests
+import json
+import six
+import time
+
+from globus_sdk.transfer.response import ActivationRequirementsResponse
+from tests.framework import CapturedIOTestCase
+
+
+class ActivationRequirementsResponseTests(CapturedIOTestCase):
+
+    def make_response(self, activated=True, expires_in=0,
+                      auto_activation_supported=True,
+                      oauth_server=None, DATA=[]):
+        """
+        Helper for making ActivationRequirementsResponses with known fields
+        """
+        data = {"activated": activated, "expires_in": expires_in,
+                "oauth_server": oauth_server, "DATA": DATA,
+                "auto_activation_supported": auto_activation_supported}
+        response = requests.Response()
+        response.headers["Content-Type"] = "application/json"
+        response._content = six.b(json.dumps(data))
+        return ActivationRequirementsResponse(response)
+
+    def test_expires_at(self):
+        """
+        Confirms expires_at is set properly by __init__
+        """
+        for seconds in [0, 10, 100, 1000, -10]:  # should -10 work here?
+            response = self.make_response(expires_in=seconds)
+            self.assertEqual(response.expires_at, int(time.time()) + seconds)
```

Comment:
Also, I don't know that the API will every actually respond with `-10`, but there's nothing saying that it can't. We should continue to include it, IMO.